### DISCUSSION TITLE
Why can't I see the "Use this template" on the manuscript-template-rstudio Github repository?

### DISCUSSION THREAD

**Discussion by Cloda07** [quarto-dev/quarto-cli#7146] [Q&A] [answered]
### Description

![Screenshot 2023-10-05 at 17 36 27](https://github.com/quarto-dev/quarto-cli/assets/42151836/5538955c-297d-4a2a-94e9-08fd2d1dce4c)

I want to follow the first step to clone the template repository, but I don't have the option (see image). Is there anything I need to change on my side? 

Thanks

**Comment by cscheid** [comment]
You can still clone the repo, but I think we just forgot to enable the "template repo" option there.

**Reply by dragonstyle** [reply, marked as answer]
Just so. I just enabled it, sorry!

**Reply by cscheid** [reply]
~~Hm, this is strange. The repository _is_ enabled as a template repo:~~

~~<img width="976" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/285675/23801127-a437-4d07-8d87-202f4526de16">~~

EDIT: Race condition, lol

**Reply by Cloda07** [reply]
Thanks!

**Reply by Cloda07** [reply]
I'm still struggling to get going with this.  I now clicked on the "GitHub to create a new repository from the template" which takes you to (https://github.com/quarto-ext/manuscript-template-rstudio/generate) but I can only create a new repository, not clone the template (see image).
![Can’t clone template](https://github.com/quarto-dev/quarto-cli/assets/42151836/314013ce-5272-4542-9b04-d990f198948d)

**Reply by mcanouil** [reply]
That's how "use template" works.
It allows you to create a new repository from the template repository which is like a fork but you don't get "upstream".

If you want to fork it, then fork it.
![image](https://github.com/quarto-dev/quarto-cli/assets/8896044/1a377485-4ba8-4b22-909e-77be7f769345)

**Reply by Cloda07** [reply]
OK, thanks. I'm just confused because this is what the tutorial example looks like. I'm not the only one that is going to get confused with this ;-)
![Create a new repository from…](https://github.com/quarto-dev/quarto-cli/assets/42151836/04b5114e-b845-4ec8-81a4-573030b3df43)

**Reply by mcanouil** [reply]
Oh I see.
So it was indeed a "clone" and "not a template".
Note that this does not change much in reality.
If you "fork" or "use template" the repository you will get a very similar thing (some UI changes from GitHub since then).

| Fork | Template |
|--------|--------|
| <img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/3ac14bb7-2d73-4bba-a8ea-aa6ca3fe6f9d"> | <img width="1624" alt="image" src="https://github.com/quarto-dev/quarto-cli/assets/8896044/f71ab56d-87a8-4368-80fb-5318b5c96b84"> |

**Reply by Cloda07** [reply]
Yes, thanks. I'm still new to git and all that, so nothing is obvious for me. I suggest the tutorial is updated to clarify that it is not an issue etc.

**Reply by mcanouil** [reply]
No worries, especially since there were UI changes, it might feel weird to not get the same wording, etc. while doing the same thing.
